apps/okreads-e2e/src/specs/search-books.spec.ts

```diff
@@ -16,12 +16,33 @@ describe('When: Use the search feature', () => {
     expect(items.length).toBeGreaterThan(1);
   });
 
-  xit('Then: I should see search results as I am typing', async () => {
+  it('Then: I should see search results as I am typing', async () => {
     await browser.get('/');
     await browser.wait(
       ExpectedConditions.textToBePresentInElement($('tmo-root'), 'okreads')
     );
 
-    // TODO: Implement this test!
+    const input = await $('input[type="search"]');
+    await input.sendKeys('javascript');
+
+    await browser.sleep(500);
+
+    let items = await $$('[data-testing="book-item"]');
+    expect(items.length).toBeGreaterThan(1);
+
+    await input.sendKeys(Key.chord(Key.CONTROL, 'a'));
```

Comment:
I couldn't get this to work on my machine (OSX), but I get what you are trying to do. Perhaps a `.clear()` would be simpler.